### ISSUE TITLE
Add-on manager doesn't install kopeio-vxlan on k8s 1.6.1

### ISSUE THREAD

**Issue by geojaz** [kubernetes/kops#2334]
kops `Version 1.6.0-alpha.1 (git-c3b794edc)` k8s `1.6.1`

k8s 1.5.6 works swimmingly (actually even with CoreOS) in private mode. Looks like we need to revisit the 1.6 stuff for CNI plugins/DNS but I think this is a known issue.
```
kops create cluster \
  --cloud aws \
  --state $KOPS_STATE_STORE \
  --node-count $NODE_COUNT \
  --zones $ZONES \
  --master-zones $MASTER_ZONES \
  --node-size $NODE_SIZE \
  --master-size $MASTER_SIZE \
  --name $CLUSTER_NAME \
  --kubernetes-version=1.6.1 \
  --networking kopeio-vxlan \
  --topology private \
  --v 10 \
  --yes
```

**Comment by geojaz** [comment]
(it just forgets that it kopeio-vxlan exists, not that there are Pod problems).

**Comment by geojaz** [comment]
Closed in #2335
